tools/pipelines/templates/build-npm-package.yml

```
@@ -139,6 +143,10 @@ variables:
   - name: toolAbsolutePath
     value:  $(Build.SourcesDirectory)/tools/telemetry-generator
     readonly: true
+  - name: testWorkspace
+    value: $(Pipeline.Workspace)/test
```

We still have to pass it as paratemer `testWorkspace` to the template for telemetry setup (we'll rename that eventually, I hope), but from this file's standpoint I think this makes the purpose of this location clearer.
```suggestion
  - name: pipelineTelemetryWorkdir
    value: $(Pipeline.Workspace)/pipelineTelemetryWorkdir
```